omnistrate_tests/test_upgrade_version.py

```
@@ -3,7 +3,11 @@
 from random import randbytes
 from pathlib import Path  # if you haven't already done so
 import threading
-
+from redis.exceptions import (
+   ConnectionError,
```

Ruff

`redis.exceptions.ConnectionError` imported but unused

Remove unused import

(F401)